features/admin.users.v1/components/user-profile.tsx

```diff
@@ -1546,21 +1552,40 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
-                                                        buttonDisableHint={ t("user:editUser." +
-                                                            "dangerZoneGroup.passwordResetZone.buttonHint") }
-                                                    />
+                                                    { isSetPassword ? (
+                                                        <DangerZone
+                                                            data-testid={ `${ testId }-set-password` }
+                                                            actionTitle={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordSetZone.actionTitle") }
+                                                            header={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordSetZone.header") }
+                                                            subheader={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordSetZone.subheader") }
+                                                            onActionClick={ (): void => {
+                                                                setOpenChangePasswordModal(true);
+                                                            } }
+                                                        />
+                                                    ) : (
+                                                        <DangerZone
+                                                            data-testid={ `${ testId }-change-password` }
+                                                            actionTitle={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordResetZone.actionTitle") }
+                                                            header={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordResetZone.header") }
+                                                            subheader={ t("user:editUser." +
+                                                                "dangerZoneGroup.passwordResetZone.subheader") }
+                                                            onActionClick={ (): void => {
+                                                                setOpenChangePasswordModal(true);
+                                                            } }
+                                                            isButtonDisabled={
+                                                                accountLocked &&
+                                                                // eslint-disable-next-line max-len
+                                                                accountLockedReason !== AccountLockedReason.PENDING_ADMIN_FORCED_USER_PASSWORD_RESET
```

Comment:
```suggestion
                                                                accountLockedReason 
                                                                !== AccountLockedReason.PENDING_ADMIN_FORCED_USER_PASSWORD_RESET
```

We should be able to break this to 2 lines

Reply:
Will fix as following 

```
accountLockedReason !== AccountLockedReason.
PENDING_ADMIN_FORCED_USER_PASSWORD_RESET
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/070b89ffe8036eac1f8ea66388f17d6f1226f175